src/commands/listProfiles.js

```diff
@@ -3,7 +3,8 @@ const { UNSUPPORTED_PROFILE } = require('../errorCodes');
 const PROFILES = require('../profiles');
 
 const listProfiles = (subcommand) => {
-  if (process.env.DISABLE_PROFILES?.toLowerCase() === 'true') {
```

Comment:
Whats the use case of accepting any value as truthy? I think it's confusing; my worry was originally that someone might pass e.g. `false` or `'false'`.

Reply:
It's not very common to pass "true" or "false" as environment variables, typically for binary options it's just set or unset/empty, or perhaps "1"/"0" (e.g. see https://no-color.org/ as an example).

I can filter out "0" and "false" explicitly, but since the default is to have it be on I don't even think it's really necessary, so I thought we should just go with the simplest thing.

Reply:
Right. Tbh, I also agree we don't really need it anymore, I'll just remove it.